src/model/Invitation.js

```diff
@@ -2,24 +2,27 @@ import Ressource from "./Ressource";
 import { getConfig } from "../config";
 import request from "../api";
 
-const _url = "/projects/:projectId/invitations";
+const _queryUrl = "/projects/:projectId/invitations";
+const _url = `${_queryUrl}/:id`;
 
 const creatableField = ["projectId", "environments", "role", "email", "force"];
 
 export default class Invitation extends Ressource {
   constructor(invitation, url) {
-    const { projectId } = invitation;
+    const { projectId, id } = invitation;
     const { api_url } = getConfig();
 
     super(
       url || `${api_url}${_url}`,
       {},
-      { projectId },
+      { projectId, id },
       invitation,
       creatableField,
       []
     );
 
+    this._queryUrl = Ressource.getQueryUrl(this._url);
```

Comment:
Why do you store this on a property and then use the global constant?

Reply:
The "global" var is a template, and the var that are using this global is static. 

This var is used in the Ressource parent class and in the function that are not static